src/components/messageSpace/messageSpace.tsx

```diff
@@ -98,19 +147,23 @@ export default function MessageSpace(props: MessageSpaceProps) {
     >
       {props.messages &&
         props.messages.length > 0 &&
-        props.messages.map((message) => {
+        props.messages.map((message, index) => {
           return (
-            <MessageCanvas
+            <div
               key={message.id}
-              message={message}
-              getActionsComponent={props.getActionsComponent}
-              getProfileComponent={props.getProfileComponent}
+              ref={index === props.messages!.length - 1 ? lastMessageRef : null}
```

Comment:
could we not add ref to the messageCanvas? why do we need to introduce a div just for this ?